### PR TITLE
Bioc release version bumps and GitHub Actions update

### PR DESCRIPTION
**Pull request by lmweber**
Hi, this PR is for:
- merging the version bumps from the Bioc maintainers from this week for the new Bioc release
- updating GitHub Actions for the latest Bioc version

### CODE REVIEW

**Comment by lmweber** [comment]
I think the GHA check error is still due to `DropletUtils` temporarily missing from the Bioc dependency package binaries. I'll check it again in the coming days or see if the GHA workflow needs to be edited.

**Comment by drighelli** [comment]
thanks for this @lmweber

**Comment by lmweber** [comment]
Ok this should be fixed now! @drighelli @HelenaLC could you please approve and then we can merge these version bumps. Thank you!

File: .github/workflows/check-bioc.yml [hunk 1]
```diff
@@ -52,9 +52,9 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - { os: ubuntu-latest, r: 'devel', bioc: '3.17', cont: "bioconductor/bioconductor_docker:devel", rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest" }
-          - { os: macOS-latest, r: 'devel', bioc: '3.17'}
-          - { os: windows-latest, r: 'devel', bioc: '3.17'}
+          - { os: ubuntu-latest, r: '4.3', bioc: '3.17', cont: "bioconductor/bioconductor_docker:RELEASE_3_17", rspm: "https://packagemanager.rstudio.com/cran/__linux__/jammy/latest" }
+          #- { os: macOS-latest, r: '4.3', bioc: '3.17'}
+          #- { os: windows-latest, r: '4.3', bioc: '3.17'}
```
Comment (drighelli) [inline .github/workflows/check-bioc.yml]: @lmweber why are osx and windows commented? 

Are they still in some process update?

**Comment by lmweber** [comment]
Hey, just bumping this. Could you please approve @drighelli or @HelenaLC ? Then we can merge these version bumps into our GitHub branches (so we don't end up with any merge conflicts during later work). Thank you!

**Comment by drighelli** [comment]
Hi @lmweber I already reviewed the updates, I was just asking why aren't we considering other platforms in the GHA

**Comment by lmweber** [comment]
Hey, I commented those out because sometimes we had errors for Windows/Mac in the GHA builds but not in the Bioconductor builds. So it was easier to remove those, since in the end only the Bioconductor builds matter for users (although the GHA builds are definitely useful for us as developers as a pre-check before pushing to Bioconductor).

I think this is resolved now though, so I'll try uncommenting them again. Thanks!

**Comment by lmweber** [comment]
Cool, looks like it worked.

If this looks ok for you could you please click approve, and I'll merge / sync all our branches / remotes. Thank you!

**Review by drighelli — APPROVED** [review]

**Comment by drighelli** [comment]
Thank you @lmweber !

File: .github/workflows/check-bioc.yml (hunk 1 above)
Comment (lmweber) [reply .github/workflows/check-bioc.yml]: Oh I didn't see this comment earlier, sorry! I never know where to find the review comments in GitHub. Thanks for approving now!

**Pull request merged**